packages/internal-workflow-common/src/workflow-options.ts

```diff
@@ -20,10 +45,10 @@ export interface BaseWorkflowOptions {
    * Specifies server behavior if a completed workflow with the same id exists. Note that under no
    * conditions Temporal allows two workflows with the same namespace and workflow id run
    * simultaneously.
-   *   ALLOW_DUPLICATE_FAILED_ONLY is a default value. It means that workflow can start if
-   *   previous run failed or was canceled or terminated.
+   *   ALLOW_DUPLICATE_FAILED_ONLY is a default value. It means that
```

Comment:
What was the intent here?

Reply:
Oops thanks—meant to move to the enum values